styles.css

```diff
@@ -1,3 +1,1317 @@
+    top: 6px;
+    border: none;
+    padding: 0;
+    cursor: pointer;
+    display: flex;
+    width: 24px;
+    height: 24px;
+    background-repeat: no-repeat;
+    background-image: url("data:image/svg+xml,%3Csvg width='15' height='10' viewBox='0 0 15 10' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M12.5 3L7.50003 8L2.5 3' stroke='white' stroke-width='3' stroke-miterlimit='10' stroke-linecap='square' stroke-linejoin='round'/%3E%3C/svg%3E ");
+}
+
+
+.nav-buttons {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    gap: 20px;
+}
+
+.breadcrumbs-container {
+    background-image: url(images/breadcrumbs-img.svg);
+    background-color: #141B22;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    width: 100%;
+    height: 242px;
+    text-align: center;
+    position: relative;
+}
+
+.breadcrumbs {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 10px;
+}
+
+.breadcrumbs-nav {
+    display: flex;
+    align-items: center;
+}
+
+.breadcrumbs-nav a {
+    text-decoration: none;
+    color: #888B8E;
+    margin: 0 10px;
+}
+
+.about-us-section {
+    box-sizing: border-box;
+    display: inline-block;
+    justify-content: center;
+    margin: auto;
+}
+
+.list-term+.list-description {
+    display: inline-block;
+}
+
+.section-text {
+    margin: auto;
+}
+
+.ot-portrait {
+    border-radius: 50%;
+
+}
+
+.list-item {
+    display: inline-block;
+    width: 250px;
+    height: 150px;
+    margin: 0 5px;
+    background: #141B22;
+    padding: 10px;
+}
+
+.out-team {
+    width: 1410px;
+    height: 824px;
+    display: inline-block;
+    align-items: center;
+
+}
+
+.ot-members {
+    padding: 0;
+}
+
+.ot-member {
+    display: inline-block;
+    justify-content: center;
+    width: 330px;
+    height: 339px;
+    background: #141B22;
+    margin: 7px;
+}
+
+.ot-portrait {
+    border-radius: 50%;
+}
+
+.partners {
+    margin: auto;
+}
+
+.joc-homepage {
+    background-color: #090C11;
+
+}
+
+.join-our-community {
+    border-radius: 8px 0px 0px 0px;
+    margin: auto;
+    background: linear-gradient(268.28deg, #2333DE -1.34%, #0ABE66 100%);
+}
+
+.partner-logo {
+    display: inline-block;
+    margin: 0 5px;
+    padding: 10px;
+}
+.bg-grey {
+    background-color: #141B22;
+}
+
+.social {
+    list-style: none;
+    display: inline-block;
+}
+
+.catalog-menu {
+    background: #141B22;
+}
+
+.colored {
+    color: #21E786;
+    margin: auto;
+    display: inline;
+}
+
+.roadmap-nav {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.phase {
+    display: flex;
+    justify-content: space-between;
+    list-style: none;
+    padding: 0;
+    width: 80%;
+    position: relative;
+}
+
+.phase:before,
+.phase-item {
+    content: '';
+    position: absolute;
+    left: 0;
+    right: 0;
+    height: 2px;
+    width: 100%;
+    background-color: #434343;
```

Comment:
For this stage, this approach is suitable, but note that the color shouldn’t be static. As it fills, the color should change.